### DISCUSSION TITLE
The plugin prometheus in kong version 1.5 doesn't has upstream metrics

### DISCUSSION THREAD

**Discussion by guinan34** [Kong/kong#9757] [Help] [answered]
i'm just very confusing why it doesn't has the upstream metric.  but in version 2.4 it's normal.  should i upgrade it to a higher version ? or may need some additional  configuration ?

**Comment by StarlightIbuki** [comment, marked as answer]
Kong 1.5 depends on https://github.com/Kong/kong-plugin-prometheus, which is now deprecated.
1.5.0 is released on 2020/01/20, while [the PR that exposes upstream health](https://github.com/Kong/kong-plugin-prometheus/pull/88) is on Jun 17, 2020. So, this is expected.
We strongly recommend using the latest version. Currently, we have 3.0.1 and 2.8.3 released.
